packages/react/src/tooltip/Tooltip.tsx

```diff
@@ -44,118 +32,29 @@ export const Tooltip = forwardRef<HTMLButtonElement, TooltipProps>(
       delayDuration,
       keepOpenOnActivation,
       onOpenChange,
-      open: openProp,
-      withArrow,
+      open,
       z = "popover",
```

Comment:
we can remove this - since `TooltipContent` already supports `z` sprinkle prop and uses `z: "popover" in the css.ts file
```suggestion
```